pkgs/os-specific/linux/ati-drivers/default.nix

```diff
@@ -1,4 +1,4 @@
-{ stdenv, fetchurl, kernel ? null, which
+{ stdenv, fetchurl, kernel ? null, linux, which
```

Comment:
`linux`? I'm confident one should use the existing `kernel` parameter instead. That can be `null`, but that's for case when a kernel module isn't built and such patches shouldn't be needed.